modules/form-field/react/stories/stories_Switch.tsx

```diff
@@ -90,3 +107,59 @@ storiesOf('Components|Inputs/Switch/React/Left Label', module)
       {control(<Switch />)}
     </FormField>
   ));
+
+storiesOf('Components|Inputs/Switch/React/Visual Testing', module)
+  .addParameters({component: Switch})
+  .addDecorator(withReadme(README))
+  .add('States', () => (
+    <StaticStates>
+      <Table>
+        <thead>
+          <tr>
+            <th></th>
+            <th>No state</th>
+            <th>Hover</th>
+            <th>Focused</th>
+            <th>Active</th>
+          </tr>
+        </thead>
+        <tbody>
+          {[false, true].map(checked => {
+            return [false, true].map(disabled => {
+              return ['Default', 'Alert', 'Error'].map(variant => {
+                const type =
+                  variant === 'Alert'
+                    ? ErrorType.Alert
+                    : variant === 'Error'
+                    ? ErrorType.Error
+                    : undefined;
+
+                const key = `${checked ? 'checked' : 'unchecked'} ${
+                  disabled ? 'disabled' : 'enabled'
+                } ${variant}`;
+
```

Comment:
@NicholasBoll made a [good point](https://github.com/Workday/canvas-kit/pull/372#discussion_r367481573) that we probably don't want to show the combination of disabled and focus/active as it may be confusing. I addressed it [here](https://github.com/Workday/canvas-kit/pull/372/commits/96551b85982dc1910136f6a893c1dbcdde69fbae) if you want to reference